src/services/database/cascadeDelete.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-param-reassign */
+import schemas from '../../schemas';
+import relationshipFinder from '../utils/relationshipFinder';
```

Comment:
This way of naming utilities is a little unconventional. It should really be called something like `getRelationships`, because then it's clear that it's going to return something.